api/app/controllers/spree/api/v2/base_controller.rb

```diff
@@ -3,6 +3,7 @@ module Api
     module V2
       class BaseController < ActionController::API
         include CanCan::ControllerAdditions
+        rescue_from ActiveRecord::RecordNotFound, with: :rescue_record_not_found
```

Comment:
I'd change the name to `record_not_found` since if you think about it this method literally doesn't rescue anything 🙄